### PR TITLE
* added json field

### PR DESCRIPTION
**Pull request by wesokes**

### CODE REVIEW

File: docs/usage_examples.rst [hunk 1]
```diff
@@ -369,6 +369,27 @@ This obviously is not an efficient query for large data sets, but it can be conv
 Joins
 -----
 
+Json Fields
+-----------
+Filtering and ordering by json fields is currently in an experimental phase.
+
+Queryset example:
+
+.. code-block:: python
```
Comment (micahhausler) [inline docs/usage_examples.rst]: You'll need a blank line between the `code-block` and the code. Currently, it won't show up https://github.com/wesokes/django-query-builder/blob/a2080d1560cdeeed2a41ee5f0652f4fde626bf59/docs/usage_examples.rst#json-fields

File: docs/usage_examples.rst [hunk 2]
```diff
@@ -369,6 +369,27 @@ This obviously is not an efficient query for large data sets, but it can be conv
...
+    from querybuilder.query import JsonQueryset
+
+    records = JsonQueryset(model=MetricRecord).filter(**{'data->field_name': 'my_value'}).order_by('data->my_sort_field')
+
+Custom field example:
+
+.. code-block:: python
+    from querybuilder.fields import JsonField
```
Comment (micahhausler) [inline docs/usage_examples.rst]: newline needed here too

**Comment by micahhausler** [comment]
Two minor docstring notes, but otherwise looks great.

I'm guessing you're using the pre 1.0 version of `django-jsonfield` because it returns a `dict` rather than a string of json. I'd recommend making a note of that in the docstring, the usage_examples, and in the changelog, so anyone who is using the `jsonfiled>=1.0` knows what they're getting.

**Comment by wesokes** [comment]
@micahhausler the json field difference might be in the way data is handled in the django model. i upgraded the test version to 1.0.2, and tests are still fine.

**Comment by micahhausler** [comment]
Oh good! Looks great!

**Comment by wesokes** [comment]
I was wrong about the json version, so I added a note in the json field examples. Going to go ahead and merge this.

**Pull request merged**